package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
+    "lint": "eslint . --ext .js,.ts",
```

Comment:
Added `lint` command to scripts

Reply:
Sure! I also added `.jsx`, `.tsx`.
It looked like it could be restricted in `react/jsx-filename-extension`, so we adopted it.
[0ad889c](https://github.com/monstar-lab-oss/reactjs-boilerplate/pull/29/commits/0ad889c71c225b320501816c02f72993de31634d), [960415e](https://github.com/monstar-lab-oss/reactjs-boilerplate/pull/29/commits/960415e1643c3ab2b859a393db9eed41febaa527)